src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java

```diff
@@ -55,12 +55,15 @@
 import static io.github.moulberry.notenoughupdates.overlays.SlayerOverlay.timeSinceLastBoss2;
 
 public class ChatListener {
+
 	private final NotEnoughUpdates neu;
-	private static final Pattern SLAYER_XP = Pattern.compile(
+
+	private static final Pattern slayerExpPattern = Pattern.compile(
```

Comment:
why did you remove the capitals? The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_").

Reply:
Crystal Hollows Overlay and Custom AH classes uses the other format too. so we change them everywhere ig